### PR TITLE
SIP-392: Support Ethena USDe as LP Collateral

### PR DESCRIPTION
**Pull request by noisekit**

### CODE REVIEW

**Comment by kaleb-keny** [comment]
probably best @noisekit to resolve the current issues , before proceeding with more deploys

**Comment by noisekit** [comment]
> probably best @noisekit to resolve the current issues , before proceeding with more deploys

yeah def.

**Comment by kaleb-keny** [comment]
Seems like the changes do a lot more than just adding ethena USDe?

**Comment by noisekit** [comment]
> Seems like the changes do a lot more than just adding ethena USDe?

There are updates for tests. And also some minor updates of arb sepolia to bring it closer to the arb mainnet (remove perps market mainly)

File: tomls/omnibus-arbitrum-mainnet/collaterals/USDe.toml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+[invoke.CoreProxy_setPoolCollateralConfiguration_USDe]
+target = ["system.CoreProxy"]
+fromCall.func = "getPoolOwner"
+fromCall.args = ["<%= extras.spartan_council_pool_id %>"]
+func = "setPoolCollateralConfiguration"
+args = [
+    "<%= extras.spartan_council_pool_id %>",
+    "<%= settings.USDe_address %>",
+    { collateralLimitD18 = "<%= settings.USDe_max_collateral_limit %>", issuanceRatioD18 = 0 },
```
Comment (kaleb-keny) [inline tomls/omnibus-arbitrum-mainnet/collaterals/USDe.toml:38]: should issuanceRatioD18 be zero here?

**Review by kaleb-keny — APPROVED** [review]

File: tomls/omnibus-arbitrum-mainnet/collaterals/USDe.toml (hunk 1 above)
Comment (kaleb-keny) [reply tomls/omnibus-arbitrum-mainnet/collaterals/USDe.toml:38]: am pretty sure it is, but not sure why 😅

Comment (noisekit) [reply tomls/omnibus-arbitrum-mainnet/collaterals/USDe.toml:38]: yes. When it is zero for collateral config, then default global value is used.

**Comment by kaleb-keny** [comment]
@noisekit 

> > Seems like the changes do a lot more than just adding ethena USDe?
> 
> There are updates for tests. And also some minor updates of arb sepolia to bring it closer to the arb mainnet (remove perps market mainly)

https://github.com/Synthetixio/synthetix-deployments/tree/main/tomls/omnibus-arbitrum-mainnet-arbthetix

But this seems to affect mainnet? The reward toml files that are not USDe

**Comment by noisekit** [comment]
> But this seems to affect mainnet? The reward toml files that are not USDe

Yes, I've just moved configs into separate files instead of keeping them in the main toml. No txns to execute.

File: omnibus-arbitrum-sepolia.toml [hunk 2]
```diff
@@ -278,91 +308,3 @@ args = [
     18,
     "<%= imports.system.contracts.USDRouter.address %>",
 ]
-
-[provision.RewardsDistributor_SpartanCouncilPool_ARB_rewards_for_ARB_lp]
```
Comment (noisekit) [inline omnibus-arbitrum-sepolia.toml:282]: @kaleb-keny better comment on code itself so we can have proper reply threads.

These configs were extracted into separate files as we were getting way to much of them at this point. This does not create any new transactions.

File: tomls/omnibus-arbitrum-mainnet/collaterals/USDe.toml (hunk 1 above)
Comment (kaleb-keny) [reply tomls/omnibus-arbitrum-mainnet/collaterals/USDe.toml:38]: Should we have default value  or the sccp value?

Comment (noisekit) [reply tomls/omnibus-arbitrum-mainnet/collaterals/USDe.toml:38]: the "default" is a global value defined by `invoke.CoreProxy_configureCollateral_USDe` 
```
issuanceRatioD18 = "<%= settings.USDe_issuance_ratio %>"
```
on line 24 above. And it is as per SCCP - `200%`

This is a specific value for this collateral for pool 1, that does not need to override global value for this collateral.

Comment (kaleb-keny) [reply tomls/omnibus-arbitrum-mainnet/collaterals/USDe.toml:38]: ohhh ok, i thought you meant by global, as in 1 value that rules them all... this is more of a local value, if it's specific to the USDe collateral i think 🤔

**Pull request merged**